R/splat-simulate.R

```diff
@@ -907,7 +907,7 @@ bridge <- function (x = 0, y = 0, N = 5, n = 100, sigma.fac = 0.8) {
     sigma2 <- runif(1, 0, sigma.fac * mean(c(x, y)))
     X <- c(0, cumsum(rnorm(N - 1, sd = sigma2) * sqrt(dt)))
     BB <- x + X - t * (X[N] - y + x)
-    BB <- akima::aspline(BB, n = n)$y
+    BB <- stats::spline(BB, n = 100, method = "natural")$y
```

Comment:
This needs to be `n = n` here, possibly why the checks failed.

Reply:
Sorry about that; should be corrected now.